Assets/MRTK/Core/Inspectors/Profiles/MixedRealityToolkitConfigurationProfileInspector.cs

```diff
@@ -131,6 +135,27 @@ protected override void OnEnable()
             {
                 renderProfileFuncs = new Func<bool>[]
                 {
+                    () => {
+                        bool changed = false;
+                        using (var c = new EditorGUI.ChangeCheckScope())
+                        {
+                            // Experience configuration
+                            if(!mrtkConfigProfile.ExperienceSettingsProfile.IsNull())
+                            {
+                                ExperienceScale experienceScale = mrtkConfigProfile.ExperienceSettingsProfile.TargetExperienceScale;
+                                string scaleDescription = GetExperienceDescription(experienceScale);
```

Comment:
I moved the inspector helpbox to the bottom to represent a high level summary of the current experience settings. I don't want to create another custom inspector for this profile because we're relatively bloated with editor/inspector dedicated files and classes.